corehq/apps/enterprise/interface.py

```diff
@@ -0,0 +1,205 @@
+    base_template = "accounting/report_filter_actions.html"
+    section_name = "Enterprise"
+    dispatcher = EnterpriseReportDispatcher
+    name = "SMS Detailed Report"
+    description = """This is a report of SMS details that can be altered by using the filter options
+    above. Once you are happy with your filters, simply click Apply."""
+    slug = "Enterprise"
+    ajax_pagination = True
+    exportable = True
+    exportable_all = True
+    export_format_override = Format.UNZIPPED_CSV
+    fields = [
+        'corehq.apps.smsbillables.interface.DateSentFilter',
+        'corehq.apps.accounting.interface.DateCreatedFilter',
+        'corehq.apps.smsbillables.interface.ShowBillablesFilter',
+        'corehq.apps.enterprise.interface.EnterpriseDomainFilter',
+        'corehq.apps.smsbillables.interface.HasGatewayFeeFilter',
+        'corehq.apps.smsbillables.interface.GatewayTypeFilter',
+    ]
+
+    @property
+    def headers(self):
+        return DataTablesHeader(
+            DataTablesColumn(_("Date of Message")),
+            DataTablesColumn(_("Project Space")),
+            DataTablesColumn(_("Direction")),
+            DataTablesColumn(_("SMS parts")),
+            DataTablesColumn(_("Gateway"), sortable=False),
+            DataTablesColumn(_("Gateway Charge"), sortable=False),
+            DataTablesColumn(_("Usage Charge"), sortable=False),
+            DataTablesColumn(_("Total Charge"), sortable=False),
+            DataTablesColumn(_("Message Log ID"), sortable=False),
+            DataTablesColumn(_("Is Valid?"), sortable=False),
+            DataTablesColumn(_("Date Created")),
+        )
+
+    @property
+    def sort_field(self):
+        sort_fields = [
+            'date_sent',
+            'domain',
+            'direction',
+            'multipart_count',
+            None,
+            None,
+            None,
+            None,
+            None,
+            'date_created',
+        ]
+        sort_index = int(self.request.GET.get('iSortCol_0', 1))
+        field = sort_fields[sort_index]
+        sort_descending = self.request.GET.get('sSortDir_0', 'asc') == 'desc'
+        return field if not sort_descending else '-{0}'.format(field)
+
+    @property
+    def shared_pagination_GET_params(self):
+        return DateSentFilter.shared_pagination_GET_params(self.request) + \
+            DateCreatedFilter.shared_pagination_GET_params(self.request) + [
+                {
+                    'name': DateCreatedFilter.optional_filter_slug(),
+                    'value': DateCreatedFilter.optional_filter_string_value(self.request)
+                },
+                {
+                    'name': ShowBillablesFilter.slug,
+                    'value': ShowBillablesFilter.get_value(self.request, self.domain)
+                },
+                {
+                    'name': NameFilter.slug,
+                    'value': NameFilter.get_value(self.request, self.domain)
+                },
```

Comment:
It looks like this report removed `corehq.apps.accounting.interface.NameFilter` from its `fields` property. Should that still be in-place, or should this code be removed?